handlers.go

```diff
@@ -75,7 +76,36 @@ func receiveHandler(producer *kafka.Producer, serializer Serializer) func(c *gin
 					Value:          metric,
 				}, nil)
 
+				go func() {
+					for event := range producer.Events() {
+						switch ev := event.(type) {
+						case *kafka.Message:
+							message := ev
+							if message.TopicPartition.Error != nil {
+								logrus.WithError(message.TopicPartition.Error).Errorf("failed to deliver message: %v",
+									message.TopicPartition)
+							} else {
+								logrus.Debugf("delivered to topic %s [%d] at offset %v",
+									*message.TopicPartition.Topic,
+									message.TopicPartition.Partition,
+									message.TopicPartition.Offset)
+							}
+						case kafka.Error:
+							logrus.WithError(err).Errorf("Error: %v", ev)
+						default:
+							logrus.Infof("Ignored event: %s", ev)
+						}
+					}
+				}()
+
```

Comment:
This function that will increase goroutines when server receives requests should move to somewhere such as main().